docs/manual/developer/07_understanding_build_system.md

```diff
@@ -74,55 +74,49 @@ and testing targets. While the specifics should be understood from this file
 directly, in general this takes the following outline of steps in rough order
 of occurrence:
 
- 1. Generate SCE content and metadata,
- 2. Generate a shorthand XML from various YAML files (like rules, variables,
-    and profiles) -- this is a single document that resembles the structure
-    of the XCCDF but isn't necessarily standards compliant,
- 3. Generation of templated content (such as Bash, OVAL, and Ansible) from
-    the metadata contained in the rule YAML files,
- 4. Transformation of the shorthand into unlinked and linked XCCDF documents,
- 5. Combining the various OVAL, OCIL, and XCCDF documents into a single
-    Datastream document,
- 6. Generating any derived products (such as CentOS and Scientific Linux).
-
+- Generate SCE content and metadata.
```

Comment:
Was it your intent to convert this to an unordered list? If not, you replace `-` with `1.`. Markdown will auto number them.

Reply:
It wasn't but it's what I understood you wanted. :confused: